Source/WebCore/css/FontFaceSet.cpp

```diff
@@ -166,6 +167,32 @@ void FontFaceSet::load(const String& font, const String& text, LoadPromise&& pro
     for (auto& face : matchingFaces)
         face.get().load();
 
+    if (scriptExecutionContext() && scriptExecutionContext()->isDocument() && downcast<Document>(scriptExecutionContext())->quirks().shouldEnableFontLoadingAPIQuirk() && !scriptExecutionContext()->settingsValues().downloadableBinaryFontsEnabled) {
```

Comment:
`scriptExecutionContext() && scriptExecutionContext()->isDocument()` could be written as `is<Document>(scriptExecutionContext())`, which is a bit more concise.